### DISCUSSION TITLE
Request to put sponsor logo on front page

### DISCUSSION THREAD

**Discussion by imbhargav5** [eslint/eslint#18035] [General] [answered]
Hi 👋, Nice to meet you all! 

My organization [Nextbase Starter Kit](https://github.com/usenextbase) is now happy to sponsor eslint organization as we use it extensively in our boilerplates and it is an amazing tool 😍. 

We started sponsoring a few days ago in the bronze tier. I was hoping to see the logo and link appear on the homepage of eslint.org as that is one of the benefits of the bronze tier. It will help with brand awareness, SEO and other business aspects for me and my small team. 

Kindly let me know if there are any steps for me to take to make this easy for the core team.
Best,
Bhargav

**Comment by kecrily** [comment]
Thank you for your sponsorship. It's auto-updated and [the code for it is here](https://github.com/eslint/eslint.org/blob/main/tools/fetch-sponsors.js). @nzakas Can you check the situation? I can't see the results of the GitHub API.

**Comment by nzakas** [comment]
That is very strange. The logos are updated automatically every night by pulling data from GitHub and Open Collective. I see you as a sponsor on GitHub, so something is up. Let me take a look.

**Reply by nzakas** [reply]
Okay, so it looks like the problem is that we are fetching the first 100 sponsors! Your sponsorship comes after that, so we're just not pulling it. I just need to update the logic in the script to continue fetching past 100 sponsors and we should be all set.

**Reply by nzakas** [reply]
Tracking issue:
https://github.com/eslint/eslint.org/issues/514

**Comment by mdjermanovic** [comment, marked as answer]
It's fixed now: https://eslint.org/sponsors/

**Reply by imbhargav5** [reply]
Thank you everyone 🤩
